libobs/util/profiler.c

```diff
@@ -1058,7 +1058,11 @@ bool profiler_snapshot_dump_csv_gz(const profiler_snapshot_t *snap,
 
 	profiler_snapshot_dump(snap, dump_csv_gzwrite, gz);
 
+#ifdef _WIN32
 	gzclose_w(gz);
+#else
+	gzclose(gz);
```

Comment:
Why is this being changed? gzclose_w is part of the public zlib interface and available on all platforms.

Reply:
The base zlib in OpenBSD is too old and doesn't have `gzopen_w()`/`gzclose_w()`. Also, it seems logical to pair `gzopen()` with `gzclose()`, and not with `gzclose_w()`, see the whole context.